public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -117,6 +126,14 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
     }
   }, [detector]);
 
+  const handleSwitchToConfigurationTab = useCallback(() => {
+    setDetecorDetailModel({
```

Comment:
small typo: rename to `setDetectorDetailModel`

Reply:
Sure, will change